tensorrt_llm/_torch/auto_deploy/models/custom/modeling_step3p7.py

```
@@ -0,0 +1,680 @@
+        getattr(config, "swiglu_limits_shared", None)
+    )
+
+
+# ---------------------------------------------------------------------------
+# Model
+# ---------------------------------------------------------------------------
+
+
+class Step3p7PreTrainedModel(PreTrainedModel):
+    base_model_prefix = "model"
+    _no_split_modules = ["Step3p7DecoderLayer"]
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)